docroot/modules/humsci/hs_siteimprove/hs_siteimprove.module

```diff
@@ -0,0 +1,6 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for stanford_siteimprove module.
```

Comment:
Is this file still necessary? And should it say `hs_siteimprove` instead of `stanford_siteimprove`?